shared/interop/generate_genesis_state.go

```diff
@@ -43,6 +46,9 @@ func GenerateGenesisState(genesisTime, numValidators uint64) (*pb.BeaconState, [
 		return nil, nil, errors.Wrap(err, "could not generate deposits from the deposit data provided")
 	}
 	root := trie.Root()
+	if genesisTime == 0 {
+		genesisTime = uint64(time.Now().Unix())
```

Comment:
can you use our roughtime package instead, over here ?
https://github.com/prysmaticlabs/prysm/blob/master/shared/roughtime/roughtime.go#L92

Reply:
Updated.